osquery/tables/system/uptime_utils.cpp

```diff
@@ -0,0 +1,29 @@
+#include <osquery/tables.h>
+
+#include "osquery/tables/system/uptime_utils.h"
+
+namespace osquery {
+namespace tables {
+
+Row secondsToTimeRow(long total_seconds) {
+  Row r;
+  r["days"] = INTEGER(total_seconds / 60 / 60 / 24);
+  r["hours"] = INTEGER((total_seconds / 60 / 60) % 24);
+  r["minutes"] = INTEGER((total_seconds / 60) % 60);
+  r["seconds"] = INTEGER(total_seconds % 60);
+  r["total_seconds"] = INTEGER(total_seconds);
+  return r;
+}
+
+QueryData genUptimeRow(long uptime_in_seconds) {
```

Comment:
this won't be necessary if all of the platform specific switching occurs in `getUptime`